src/utilities/util_printlog.c

```diff
@@ -49,7 +75,8 @@ util_printlog(WT_SESSION *session, int argc, char *argv[])
     if (argc != 0)
         return (usage());
 
-    if ((ret = __wt_txn_printlog(session, ofile, flags)) != 0)
+    if ((ret = __wt_txn_printlog(session, ofile, flags, start_set == true ? &start_lsn : NULL,
+           end_set == true ? &end_lsn : NULL)) != 0)
         (void)util_err(session, ret, "printlog");
 
     return (ret);
```

Comment:
(Not related to this line, just adding this at the end of the diff.)
There needs to be some tests added for this functionality. Maybe look at `test/suite/test_tnx08.py` and add something new that just tests this functionality with printlog. It should test valid and invalid combinations of LSNs.

Reply:
Yeah, definitely a good idea!